sdk/identity/Azure.Identity/src/Credentials/AzurePowerShellCredential.cs

```diff
@@ -264,7 +265,7 @@ private void GetFileNameAndArguments(string resource, string tenantId, out strin
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 fileName = Path.Combine(DefaultWorkingDirWindows, "cmd.exe");
-                argument = $"/d /c \"{powershellExe} \"{commandBase64}\" \"";
+                argument = $"/d /c \"{powershellExe} \"{commandBase64}\" \" & exit";
```

Comment:
```suggestion
                argument = $"/d /c \"{powershellExe} \"{commandBase64}\" \" && exit";
```

But why is this necessary? If the app fails or succeeds, it'll still exit itself. Why the separate shell `exit`?

Reply:
This allows the error status from the internal command being run by cmd.exe to propagate out. Before this change, if pwsh wasn't found we'd get the generic status 1 back. After this change we get back 9009 (file not found).

Reply:
I'm still puzzled as to why. The shell (cmd) is the thing setting the process exit code, but you're saying it behaves differently with and without the explicit `exit` command (not process)? I'll sign off and believe you, but I'm worried it's a behavior observation and not necessarily guaranteed. It's something to watch.